### DISCUSSION TITLE
Proposal: Rename tag C-good first issue to good first issue

### DISCUSSION THREAD

**Discussion by Xuanwo** [databendlabs/databend#5455] [Show and tell]
Github has native good first issue support which makes the `good first issue` label special. 

For example

https://github.com/datafuselabs/opendal/contribute

![image](https://user-images.githubusercontent.com/5351546/169188055-36be24ef-4155-42f7-97d5-45278218a69f.png)

Compared to

https://github.com/datafuselabs/databend/contribute

![image](https://user-images.githubusercontent.com/5351546/169188087-b598acae-bf0c-42a9-89c4-cc9792dce79b.png)

---

Resolved

![image](https://user-images.githubusercontent.com/5351546/169191299-8748f56c-6cdd-47a2-940f-906bf9c30548.png)

**Comment by Xuanwo** [comment]
Ping @BohuTANG for review.

**Comment by TCeason** [comment]
Maybe we also can add our contribution in the future.


https://databend.rs/doc/contributing
